content/blog/refinements-in-ruby.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Refinements in Ruby: A Flexible Approach to Modifying Core Classes"
+slug = "refinements-in-ruby"
+date = 2024-09-13T11:48:18+05:30
+image = "/images/2024/refinements-in-ruby/header.jpeg"
+draft = false
+authors = ["Syed Mohd Mehndi"]
+description = "Refinements in Ruby: A Flexible Approach to Modifying Core Classes"
+tags = ["Ruby"]
+categories = ["Ruby"]
```

Comment:
Looking at some of our other tags and adding the ones that are relevant for this will be great, we should aim for at least 3

Reply:
That helps with visibility

Reply:
Added "Clean Code" tag along with "Ruby"
Third tag is "Ruby on Rails" which is commonly used in RoR acticles, but this article only talks about ruby, so have not used RoR tag here